sorc/link_workflow.sh

```diff
@@ -137,29 +145,43 @@ fi
 #------------------------------
 #--add GSI fix directory
 #------------------------------
-cd ${pwd}/../fix                ||exit 8
+if [ -d ../sorc/gsi_enkf.fd ]; then
+  cd ${pwd}/../fix                ||exit 8
     [[ -d fix_gsi ]] && rm -rf fix_gsi
     $LINK ../sorc/gsi_enkf.fd/fix  fix_gsi
+fi
 
+#------------------------------
+#--add GDASApp fix directory
+#------------------------------
+if [ -d ../sorc/gdas.cd ]; then
+  cd ${pwd}/../fix                ||exit 8
+    [[ -d fix_gdas ]] && rm -rf fix_gdas
+    $LINK $FIX_DIR/fix_gdas .
+fi
 
 #------------------------------
-#--add GDASApp files
+#--add GDASApp files 
```

Comment:
Extra space was added at the end of this line

Reply:
Extra space removed at [53befc3](https://github.com/NOAA-EMC/global-workflow/commit/53befc3d11a5b6e8ff5404e8eed2ecd48a6bc0bc)